### PR TITLE
typos suggestion

### PR DESCRIPTION
**Pull request by ccoVeille**
## Check list

- [X] I have performed a self-review of my code
- [ ] I have commented my code in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

## Description
- **Fix some typos and wordings in README.md**
- **Fix headers style**
- **Fix exemplī grātiā usage (Latin)**

## Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Breaking change
- [X] Documentation change

## Test environment

- Shell
    - [ ] bash
    - [ ] zsh
    - [ ] fish
- OS
    - [ ] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Others:

### CODE REVIEW

**Review by sandr01d — APPROVED** [review]
LGTM

**Review by wfxr — APPROVED** [review]
Thanks @ccoVeille. Mostly LGTM, except for a minor comment.

File: README.md [hunk 1]
```diff
@@ -328,18 +328,18 @@ export FORGIT_LOG_FZF_OPTS='
 '
 ```
 
-## other options
+## Other options
```
[the remark this answers is not in the document]
Comment (ccoVeille) [reply README.md:331]: I'll change it and push once I'll be back in front of a computer.

[the remark this answers is not in the document]
Comment (ccoVeille) [reply README.md]: pushed back

**Comment by carlfriedrich** [comment]
@ccoVeille Thanks for your contribution, looks good to me.

**Pull request merged**